src/Essentials/src/Permissions/Permissions.ios.cs

```diff
@@ -249,6 +250,47 @@ public override Task<PermissionStatus> RequestAsync()
 			}
 		}
 
+		public partial class PostNotifications : BasePlatformPermission
+		{
+			/// <inheritdoc/>
+			#pragma warning disable RS0016 
```

Comment:
We'll want to remove this and add the right entries to the API txt files. That will also mean we want to target this for .NET 10 as it adds new APIs